teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super(id, age)
+    @specialization = specialization
+  end
```

Comment:
- [ ] The `Student constructor`  extends parent's constructor by adding `@specialization` and a parameter for it, checkout this [link](`https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars`) for more clarification on that. Something like so;
```ruby
  def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```